README.md

```diff
@@ -674,6 +674,7 @@ Services to securely store your Docker images.
 # Useful Resources
 
 - **[Valuable Docker Links](http://nane.kratzke.pages.mylab.th-luebeck.de/about/blog/2014/08/24/valuable-docker-links/)** High quality articles about docker! **MUST SEE**
+- [Become a Docker Power User with Visual Studio Code](https://www.thebyte.io/become-docker-power-user-with-vs-code)
```

Comment:
Since it is behind a paywall, could you add :heavy_dollar_sign: `heavy_dollar_sign` to it ?

Reply:
Done ✅